staff/miguel-angel-zapatero/cv/index.html

```diff
@@ -0,0 +1,94 @@
+        </section>
+    </header>
+    <article>
+        <section>
+            <h1>FORMACIÓN</h1>
+            <ul>
+                <li>
+                    <hgroup>
+                        <h2>Skylab Coders Academy</h2>
+                        <h3>Desarrollador Web Full Stack???</h3>
+                    </hgroup>
+                    <p>ABRIL DEL 2019 - JULIO DEL 2019, BARCELONA</p>
+                    <p>Full-Time Bootcamp (+500h) - Javascript, HTML, CSS, Git, TDD, jQuery, ReactJS, NodeJS, ExpressJS, API’s y MongoDB.</p>
+                </li>
+                <li>
+                    <hgroup>
+                        <h2>I.E.S. "Sa Palomera"</h2>
+                        <h3>Técnico superior en Administración de Sistemas Informáticos</h3>
+                    </hgroup>
+                    <p>SEPTIEMBRE DEL 2001 - JUNIO DEL 2003, BLANES</p>
+                </li>
+            </ul>
+        </section>
+        <section>
+            <h1>EXPERIENCIA</h1>
+            <ul>
+                <li>
+                    <hgroup>
+                        <h2>Blanes Distribuidora, S.A.</h2>
+                        <h3>Departamento informático</h3>
+                    </hgroup>
+                    <p>FEBRERO DEL 2005 - FEBRERO DEL 2019, BLANES</p>
+                    <ul>
+                        <li>Responsable de la infraestructura del HW, SW y redes (LAN, WAN y WIFI)</li>
+                        <li>Control de stocks, albaranes, pedidos y temas administrativos¿?¿?¿?¿ </li>
+                        <li>Logros: Cambio del programa de gestión y migración al nuevo HW (servidor y PCs)</li>
+                    </ul>
+                </li>
+            </ul>
+        </section>
+        <section>
+            <h1>INFORMATICA</h1>
+            <ul>
+                <li>Conocimiento de los productos QlikView y Qlik Sense para la analítica y gestión de datos en Business Intelligence.</li>
+                <li>Conocimiento de Lenguajes de Programación (PHP), CMS (Joomla, Wordpress) y  MySQL.</li>
+                <li>Conocimiento completo de Microsoft Office.</li>
+                <li>Usuario de Internet, familiarizado en las últimas tendencias en la red y redes sociales. (poner esto?¿)</li>
+            </ul>
+        </section>
+        <aside>
+            <section>
+                <h1>APTITUDES</h1>
+                <p>Trabajar en grupo</p>
+                <p>Hacer mimitos a mi novia :P</p>
+                <p>Muchas cosas mas...</p>
+            </section>
+            <section>
+                <h1>IDIOMAS</h1>
+                <p> Castellano (nativo)</p>               
+                <p>Catalán (nativo)</p>
+                <p>Inglés (First Certificate Diciembre del 2014)</p>
+            </section>
+            <section>
+                <h1>INTERESES</h1>
+                <p>Escalada y montaña</p>
+                <p>Tecnología y videojuegos</p>
+            </section>
+        </aside>
+    </article>
+    <footer></footer>
```

Comment:
@maz-e it makes no sense to open and close a tag with no data inside. Please fill or remove. Thanks!

Reply:
@lexbarba footer removed! 👍